app_test.go

```diff
@@ -64,6 +64,18 @@ func NewSpied(opts ...Option) (*App, *fxlog.Spy) {
 	return New(opts...), spy
 }
 
+func NewValidateAppErr(tb testing.TB, opts ...Option) error {
```

Comment:
minor: would you please rename this to validateApp or validateTestApp, since the net effect of this is to validate the provided application (comprised of the options) with ValidateApp?

(If this returned the App, then the `New*` would work better, but the current behavior is preferable for the use case.)